frontend/src/components/custom-tools/prompt-card/PromptCard.jsx

```diff
@@ -613,22 +627,13 @@ function PromptCard({
 
     if (profileManagerId) {
       body.profile_manager = profileManagerId;
-      let tokenUsageId;
       let url = `/api/v1/unstract/${sessionDetails?.orgId}/prompt-studio/fetch_response/${details?.tool_id}`;
       if (!isSimplePromptStudio) {
         body["run_id"] = runId;
-        tokenUsageId = getTokenUsageId(promptId, docId, profileManagerId);
```

Comment:
@tahierhussain Minor doubt : Will there be an impact in invoking token usage api?

Reply:
@harini-venkataraman We no longer have to call the token usage API. We get the token usage details along with the prompt output API's response itself. Hope this answers your question.